### DISCUSSION TITLE
I just used sh1mmer to unblock dev mode, but my dumbass dosent even know how to turn on dev mode

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#513] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hi

**Comment by Aka-but-nice** [comment]
Lmfao

**Comment by Aka-but-nice** [comment]
Which one, the chrome://extensions one, or the one where your entire device goes into dev mode?

**Reply by Aka-but-nice** [reply]
Both I listed have tons of tutorials online about how to do it

**Comment by obkduishinijf** [comment, marked as answer]
chrome://extensions lets you sideload extensions, but for full dev mode on the chromebook, turn it off, then on, and WHILE it's turning back on, hold esc, power, and refresh. hold ctrl and d, then continue through it. that's how you fully turn on dev mode for the chromebook.

**Reply by funkymokey** [reply]
which one does sh1mmer unblock

**Reply by obkduishinijf** [reply]
> which one does sh1mmer unblock

sh1mmer should unblock both, unless you don't unenroll, then only full dev mode is accessible.
